cli/flox/src/commands/pull.rs

```diff
@@ -643,8 +644,9 @@ mod tests {
     /// Pulling an environment without packages for the current platform should
     /// prompt about adding system
     /// When the user confirms, the environment should not be removed
-    #[test]
-    fn test_handle_pull_result_4() {
+    // handle_pull_result() calls spin() which depends on tokio
```

Comment:
Ouch i didnt realize that these tests actually spin, that's calling for some weird output when this test fails some day ^^

(quib)